test/e2e/webhook_test.go

```diff
@@ -77,11 +76,123 @@ var _ = Describe("bundle api validating webhook", func() {
 		})
 		AfterEach(func() {
 			By("deleting the testing Bundle resource for failure case")
-			err := c.Delete(ctx, bundle)
-			Expect(err).To(WithTransform(apierrors.IsNotFound, BeTrue()))
+			err = c.Delete(ctx, bundle)
+			Expect(err).ToNot(BeNil())
```

Comment:
Can we assert on something specific in the error?

Generic `err != nil` assertions could keep passing if we make an unwanted change that results in a different error.

Reply:
I'll put the `Expect(err).To(WithTransform(apierrors.IsNotFound, BeTrue()))` back.  Thanks!